lib/src/commands/fetch.dart

```diff
@@ -121,7 +139,7 @@ class FetchCommand extends BaseCommand {
       print('(Max processed package count of $maxCount reached.)');
     }
 
-    print(count);
+    print('---');
```

Comment:
IIRC printing the count was useful to get some sense for overall progress but maybe that's redundant now somehow?

Reply:
Yes, but it was printed 3x and it bugged me a little bit; this is what it looks like now:

```
Skipped package: marker_prof (does not use flutter)
Skipped package: handle_system (does not use flutter)
9159 packages processed
(Processed all available packages on pub.)
---
Creating: third_party/download
Extracted third_party/cache/basic_components-0.0.4+3
Installing dependencies for basic_components
```

Reply:
LGTM!